src/main/scala/com/databricks/labs/overwatch/pipeline/Bronze.scala

```diff
@@ -35,6 +35,10 @@ class Bronze(_workspace: Workspace, _database: Database, _config: Config)
       BronzeTargets.jobRunsSnapshotTarget
     )
   }
+  def targetToSnap: Array[PipelineTable] = {
+    getAllTargets :+ pipelineStateTarget
+  }
+
```

Comment:
don't put this here -- and definitely don't make it public. targetsToSnap should be a class variable inside of Snapshot class.